src/actions/types.ts

```diff
@@ -70,6 +70,13 @@ export type DelegateActionParams = {
   delegator?: Address
 }
 
+export type VoteActionParams = {
```

Comment:
proposalId and support are parameters to the vote functions on the contracts.

Just to confirm, we pull these params from the functions, not the events those functions emit right?